JDStatusBarNotification/JDStatusBarView.m

```diff
@@ -92,4 +107,32 @@ - (CGSize)currentTextSize;
   return textSize;
 }
 
++ (BOOL)isIphoneX {
+    NSString *modelName = [self modelName];
+    return [modelName isEqualToString: @"Phone10,3"] || [modelName isEqualToString: @"iPhone10,6"] || [JDStatusBarView isSumulatorIphoneX];
+}
+
++ (NSString *)modelName {
+    struct utsname systemInfo;
+    uname(&systemInfo);
+    return [NSString stringWithCString:systemInfo.machine encoding:NSUTF8StringEncoding];
+}
+
++ (BOOL)isSumulatorIphoneX {
+    if ([JDStatusBarView isSimulator] && [UIScreen mainScreen].bounds.size.height == 812) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
++ (BOOL)isSimulator {
```

Comment:
let's keep that as a macro, no need to ship a method that always returns NO in prod.